models/yang/sonic/sonic-mirror-session.yang

```diff
@@ -54,6 +54,18 @@ module sonic-mirror-session {
 				leaf queue {
 					type uint8;
 				}
+
+				leaf dst_port {
+					type string;
+				}
+
+				leaf src_port {
+					type string;
+				}
+
+				leaf direction {
+					type string;
```

Comment:
Can you define as enum: "rx", "tx" and "both"? It can be used for monitoring ingress, egress or both packets.